### PR TITLE
Update TLDs to 2016053000

### PR DESCRIPTION
**Pull request by vklquevs**
From https://data.iana.org/TLD/tlds-alpha-by-domain.txt

Also includes a quick and dirty script for future updates.

### CODE REVIEW

File: regenerate-tlds.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+list=tlds-alpha-by-domain.txt
+
+rm -f $list
+wget http://data.iana.org/TLD/$list
```
Comment (ghost) [inline regenerate-tlds.sh]: should use https:

**Comment by ghost** [comment]
This LGTM, thanks! Bonus points for the script :)

**Comment by ghost** [comment]
@jbenet can I have collab?

**Comment by fbaiodias** [comment]
Ping @jbenet & @lgierth, I'm not being able to use IPNS with a `.site` domain and it looks like this PR would fix it.
Can you please have a look? :octocat:

**Comment by jbenet** [comment]
sorry for delay everyone.

@lgierth you have collab. 

This PR LGTM! (i like the generation approach, `sh` or `go generate`).

later on, we should add a function with a regex to capture TLDs that dont exist yet. it wont be perfect, but hey.

**Comment by Stebalien** [comment]
(has replacement PR)

**Pull request closed without merging**